binding/springjms/src/main/java/io/tracee/binding/springjms/JmsHelper.java

```diff
@@ -0,0 +1,49 @@
+package io.tracee.binding.springjms;
+
+import io.tracee.Tracee;
+import io.tracee.TraceeBackend;
+import io.tracee.TraceeConstants;
+import io.tracee.Utilities;
+
+import javax.jms.JMSException;
+import javax.jms.Message;
+import java.util.Map;
+
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.AsyncDispatch;
+import static io.tracee.configuration.TraceeFilterConfiguration.Channel.AsyncProcess;
+
+/**
+ * For manual reading / writing from/to messages
+ */
+public class JmsHelper {
+
+	@SuppressWarnings("unused")
```

Comment:
`@SupressWarnings("unused")` Should not be necessary. It is exposed as public api.